packages/p2p/src/components/buy-sell/buy-sell-form.jsx

```diff
@@ -223,41 +223,50 @@ const BuySellForm = props => {
+                                                            add_payment_method={add_payment_method}
+                                                            is_add
                                                             key={key}
                                                             medium
-                                                            onClick={() => onClickPaymentMethodCard(payment_method)}
-                                                            payment_method={payment_method}
-                                                            style={
-                                                                selected_methods.includes(payment_method.ID)
-                                                                    ? style
-                                                                    : {}
-                                                            }
+                                                            onClickAdd={() => {
+                                                                my_profile_store.setSelectedPaymentMethodDisplayName(
+                                                                    add_payment_method
+                                                                );
+                                                                my_profile_store.setShouldShowAddPaymentMethodForm(
+                                                                    true
+                                                                );
+                                                            }}
                                                         />
-                                                    ))
-                                                ) : (
-                                                    <PaymentMethodCard
-                                                        add_payment_method={add_payment_method}
-                                                        is_add={true}
-                                                        key={key}
-                                                        medium
-                                                        onClickAdd={() => {
-                                                            my_profile_store.setSelectedPaymentMethodDisplayName(
-                                                                add_payment_method
-                                                            );
-                                                            my_profile_store.setShouldShowAddPaymentMethodForm(true);
-                                                        }}
-                                                    />
-                                                );
-                                            })}
+                                                    );
+                                                })
+                                                .sort(payment_method_card_node =>
```

Comment:
Added sorting to sort cards that has `is_add` to true to be rendered at the end of the list